### DISCUSSION TITLE
Exponential backoff retry

### DISCUSSION THREAD

**Discussion by motawy** [taskforcesh/bullmq#585] [Q&A]
In a scenario where there is a queue on Rabbit that posts data and if it fails, it goes onto Bull to be retried with an exponential backoff strategy. 

Now the retry is handled by Bull but since I wanted to reduce the load on the worker, I wanted to republish that message onto the Rabbit queue since the handler has already all the logic. 

Will Bull have any information of that job and be able to trace it to the first failure and continue the backoff strategy, or is there a way to manually put that information when the job is failing? Or it's not possible to achieve something like that?

**Comment by manast** [comment]
I am not sure I understand what you mean. But you can create custom backoffs, it is quite flexible so maybe it solves the requirements you have: https://docs.bullmq.io/guide/retrying-failing-jobs
